roles/sap_hana_install/tasks/post_install.yml

```diff
@@ -160,6 +160,19 @@
     __sap_hana_install_fact_hana_version: "{{ __sap_hana_install_register_install_result.stdout.split(';')[0] }}"
   when: not ansible_check_mode
 
+
```

Comment:
Please remove empty line so the spacing between tasks is similar to other tasks.